map_it_test.go

```diff
@@ -1057,6 +1057,40 @@ func TestMap_SetTTL(t *testing.T) {
 	})
 }
 
+func TestMap_SetTTLAffected(t *testing.T) {
+	it.MapTester(t, func(t *testing.T, m *hz.Map) {
+		ctx := context.Background()
+		testcases := []struct {
+			key        string
+			isEffected bool
+			expectErr  bool
```

Comment:
`Err` prefixed identifiers are usually `error`s. How about renaming that to something like `failureExpected`?

Reply:
Since we are asserting for an expected error, I do not think renaming it as "failure" is a good practice. Sadly I do not have a reference to defend this point. If I can find an example for this on standard library, I will share it

Reply:
https://github.com/golang/go/blob/master/src/crypto/x509/name_constraints_test.go#L46
This is an example from std lib.

Reply:
You are not asserting for an expected error, but the existence of an error. The field in the link you mention is a string field (presumably the contents of an error message). How about `errorExpected` then?

Reply:
That sounds good to me, thanks